### DISCUSSION TITLE
Welcome to galpy Discussions!

### DISCUSSION THREAD

**Discussion by jobovy** [jobovy/galpy#487] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by JosephKarpinski** [comment]
Well, maybe you can help debug this Python 3.12.3 code running under VSCode Jupyter Notebook on a M1 iMac.
I'm at a loss. Correct one error, another shows up. Stressed out three Ai's trying to debug this. Grok was the first to give up and simply rewrote the code without using galpy. Reference article:  https://iopscience.iop.org/article/10.3847/1538-4357/ad382d. Targeting GD-1 orbit & stream modelling

**Reply by jobovy** [reply]
Hi @JosephKarpinski 👋 

I'm happy to try to help, but (a) it would be best to open a separate Discussion topic rather than this general intro to the discussions one and (b) please include a clear description of the problem you are having. You posted a lot of code, but I have no idea what your issue with it is!

**Comment by JosephKarpinski** [comment]
Will work on it on Monday. Thanks.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by JosephKarpinski** [comment]
Worked on it Monday and Tuesday. Finally got something to work.
See attachments.

<img width="1322" height="768" alt="Screenshot 2025-11-25 at 4 12 02 PM" src="https://github.com/user-attachments/assets/f2c0a254-e23b-4db1-8ce8-c2310b566ae6" />
<img width="942" height="959" alt="Screenshot 2025-11-25 at 4 54 29 PM" src="https://github.com/user-attachments/assets/dd204dea-49cb-44bd-81cf-5546a06e83db" />
<img width="942" height="959" alt="Screenshot 2025-11-25 at 4 54 47 PM" src="https://github.com/user-attachments/assets/29bd4a4d-2db6-4659-95aa-c803404e3522" />


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by jobovy** [reply]
Hi [@JosephKarpinski](https://github.com/JosephKarpinski), GitHub doesn't include email attachments when you reply to a comment by email (which is what I assume you did), so you'll have to go the webpage to add files to your comments. But please also note that a shorter piece of code that returns the issue you are having is much easier to review than a long attachment.

**Reply by JosephKarpinski** [reply]
Deleted the code part of the  posting. Is this now posted as a separate discussion or should I delete it and try posting again.
